### PR TITLE
Web2

### PR DESCRIPTION
**Pull request by bdmason**

### CODE REVIEW

File: subschemas/web2/src/config/index.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import dotenv from 'dotenv';
+import development from './development';
+import production from './production';
+
+dotenv.config({ debug: true });
```
Comment (bdmason) [inline subschemas/web2/src/config/index.ts:5]: we shouldn't need to run `dotenv` in here, it's already done in the `api-gateway` application

File: subschemas/web2/src/config/production.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const config: any = {
```
Comment (bdmason) [inline subschemas/web2/src/config/production.ts:1]: we need to have a chat about deploying this

**Comment by bdmason** [comment]
@chenzongxiong I've added a couple of comments, the main thing is we need to talk about deployment

**Comment by chenzongxiong** [comment]
@bdmason I migrated `web2-*` and `ipfs-*` to `my-crypto-profil`. we can close this issue and delete related branches, `web2-ipfs` & `web2`

**Comment by bdmason** [comment]
👍

**Pull request closed without merging**